src/modules/remote.json

```diff
@@ -0,0 +1,156 @@
+{
+  "openrpc": "1.2.4",
+  "info": {
+    "title": "Remote",
+    "description": "A module for managing Remote keys, paring and programming.",
+    "version": "0.0.0"
+  },
+  "methods": [
+    {
+      "name": "getRemoteList",
```

Comment:
The firebolt method style is to try and have one word method names if possible and to not have phrases like "get" in them.
Remote is maybe a little redundant here because this is the remote module.
Can't we just call this method `list` or `paired`

Reply:
Agreed and updated